run_splunk_container.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+IMAGE_NAME="custom-splunk"
+
+get_random_port() {
+  echo $(( 49152 + RANDOM % (65535 - 49152 + 1) ))
+}
+
+API_PORT=$(get_random_port)
+WEB_PORT=$(get_random_port)
```

Comment:
I don't think we need to randomly assign ports since this image won't be used for parallel automated testing. The grep for `IMAGE_NAME` prevents this anyway. It'd be simpler and more predictable to let the script invoker specify these using env variables should they need to.